src/actions/favorites.js

```diff
@@ -0,0 +1,16 @@
+const FAVORITE_TOGGLE = 'favorite_toggle';
+const FAVORITE_FILTER = 'favorite_filter';
+/* Toggle Favorite */
+const toggleFavorite = (payload) => ({ type: FAVORITE_TOGGLE, payload });
+const filterFavorite = (filter) => ({ type: FAVORITE_FILTER, filter});
```

Comment:
Action creators should wrap the payload inside an object:

```
const toggleFavorite = (gif) => (
  type: FAVORITE_TOGGLE,
  payload: { filter },
);
```
This is in order to have a standard way of transporting payloads, and
comply with the Flux standard.